index.html

```diff
@@ -3016,30 +3016,56 @@ <h2>Keep Personal Data Private</h2>
       </p>
     </section>
 
+    <section>
+      <h3>Relationship to the Same-Origin Policy</h3>
+
+      <p>
+The <a href="https://en.wikipedia.org/wiki/Same-origin_policy">
+Same-origin policy</a> is a security and privacy concept that constrains
+information to the same Web domain by default. There are mechanisms, such as
+[[[?WEBAUTHN]]], that extend this policy to cryptographic keys. When a
+cryptographic key is bound to a specific domain, it is sometimes referred to
+as a <dfn>pairwise identifier</dfn>.
+      </p>
+      <p>
+This specification is closer to
+<a href="https://en.wikipedia.org/wiki/Cross-origin_resource_sharing">
+Cross-origin resource sharing</a> (CORS) both in concept and practice. The
+result of this is that correlatable information can be shared between origins
+and while that can lead to positive security outcomes (no public key
+registration burden), it can also lead to negative privacy outcomes (tracking).
+Those that use this specification are warned that there are trade-offs with
+each approach and to use the mechanism that maximizes security and privacy
+according to the needs of the individual or organization. Using a
+[=controller document=] for all use cases is not always advantageous when a
+same-origin bound cryptographic key would suffice.
+      </p>
+    </section>
+
     <section>
       <h2>Identifier Correlation Risks</h2>
 
       <p>
-Identifiers can be used for the purpose of correlation.
-[=Controllers=] can mitigate this privacy risk by using pairwise identifiers
-that are unique to each relationship; in effect, each identifier acts as a
-pseudonym. A pairwise identifier need only be shared with more than one party
-when correlation is explicitly desired. If pairwise identifiers are the default,
-then the only need to publish an identifier openly, or to share it with multiple
-parties, is when the [=controllers=] and/or [=subjects=] explicitly
-desire public identification and correlation.
+Identifiers can be used for the purpose of correlation. [=Controllers=] can
+mitigate this privacy risk by using [=pairwise identifiers=] that are unique to
+each relationship; in effect, each identifier acts as a pseudonym. A [=pairwise
```

Comment:
```suggestion
mitigate this privacy risk by using [=pairwise identifiers=] that are unique to
each relationship or context; in effect, each identifier acts as a pseudonym. A [=pairwise
```

Reply:
I'm a bit concerned that `context` may be read as `@context`, leading to various confusion. Could a synonym be substituted here (and below)?

Reply:
Hmm, I do think that's really the right word and my memory is that it's commonly used when talking about identifiers. I didn't see a good alternative immediately in a quick thesaurus search and I don't know how much we should worry that people might confuse context with `@context` such that we're not able to use both as needed.

Reply:
I updated to use "interaction domain"